components/controllers/ConsistentEvaluationHrefController.js

```
@@ -108,4 +108,32 @@ export class ConsistentEvaluationHrefController {
 			submissionType
 		};
 	}
+
+	async getGradeItemInfo() {
```

This function was written to fail silently it seems. What's the impact if any of the 3 properties are missing?

---

Yeah, fail silently since its valid for the grade item link to not be present - I've tried to make sure I handle the absence of any of the properties in the downstream components

---

I think you'll still need some of the changes from the now closed PR in grade-result to handle the absences